src/commands/basemaps-topo-import/list-jobs.ts

```diff
@@ -0,0 +1,72 @@
+import { fsa } from '@chunkd/fs';
+import { command, option, string } from 'cmd-ts';
+
+import { CliInfo } from '../../cli.info.js';
+import { logger } from '../../log.js';
+import { isArgo } from '../../utils/argo.js';
+import { config, forceOutput, registerCli, verbose } from '../common.js';
+
+type Output = {
+  output: string;
+  input: string[];
+};
```

Comment:
```suggestion
interface Output {
  output: string;
  input: string[];
};
```

this should be a type somewhere already.